### PR TITLE
Price histogram data restored

### PR DESCRIPTION
**Pull request by jitu5**
## Description

Resolves [#1568](https://github.com/kedro-org/kedro-viz/issues/1568)

In this PR there is no code change as its not a bug, Price histogram data got deleted now its been restored.

Screenshot 2023-11-08 at 12 53 38 p m
 
## Fix 
This PR also includes fix for Circle CI build fails due to recent breaking changes introduced by MUI by moving TreeView/TreeItem from "@mui/lab" to "@mui/x-tree-view".

## Checklist

- [X] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

### CODE REVIEW

**Review by tynandebold — COMMENTED** [review]
Looks good!

**Review by tynandebold — APPROVED** [review]

**Review by rashidakanchwala — APPROVED** [review]
thanks!

**Comment by jitu5** [comment]
Thanks @rashidakanchwala for help!

**Review by ravi-kumar-pilla — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -49,9 +49,9 @@
     "@emotion/styled": "^11.10.6",
     "@graphql-tools/schema": "7.1.5",
     "@mui/icons-material": "^5.11.9",
-    "@mui/lab": "^5.0.0-alpha.120",
     "@mui/material": "^5.11.10",
     "@mui/styles": "^5.11.9",
+    "@mui/x-tree-view": "^6.17.0",
```
Comment (ravi-kumar-pilla) [inline package.json:54]: awesome !!

Comment (rashidakanchwala) [reply package.json:54]: thanks!! good catch and fix !!

**Pull request merged**